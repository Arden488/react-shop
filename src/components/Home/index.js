import React from 'react';

import Slider from '../Slider';
import HotProducts from '../HotProducts';
import FeaturedProducts from '../FeaturedProducts';
import DiscountedProducts from '../DiscountedProducts';

import './styles.css';

const Home = () => (
  <div>
    <Slider />
    
    <h2>Top sale:</h2>
    <HotProducts />

    <h2>Featured</h2>
    <FeaturedProducts />

    <h2>Best discounts:</h2>
    <DiscountedProducts />
  </div>
);

export default Home;
