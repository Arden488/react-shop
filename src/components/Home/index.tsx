import * as React from 'react';

import DiscountedProducts from '../DiscountedProducts';
import FeaturedProducts from '../FeaturedProducts';
import HotProducts from '../HotProducts';
import Slider from '../Slider';

import './styles.css';

const Home = () => (
  <div>
    <Slider />
    
    <h2>Top sale:</h2>
    <HotProducts />

    <h2>Featured</h2>
    <FeaturedProducts />

    <h2>Best discounts:</h2>
    <DiscountedProducts />
  </div>
);

export default Home;
