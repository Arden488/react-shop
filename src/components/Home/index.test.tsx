/* global it, expect, shallow, beforeEach */

import React from 'react';
import Home from './index';

let component = null;

beforeEach(() => {
  component = shallow(<Home />);
});

it('renders correctly', () => {
  expect(component).toMatchSnapshot();
});
