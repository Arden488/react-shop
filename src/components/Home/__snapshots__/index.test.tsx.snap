// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2>
    Top sale:
  </h2>
  <HotProducts
    items={
      Array [
        1,
        2,
        3,
      ]
    }
  />
  <h2>
    Featured
  </h2>
  <FeaturedProducts
    items={
      Array [
        1,
        2,
        3,
      ]
    }
  />
  <h2>
    Best discounts:
  </h2>
  <DiscountedProducts
    items={
      Array [
        1,
        2,
        3,
      ]
    }
  />
</div>
`;
